### PR TITLE
Report expense analytic account

### PR DESCRIPTION
**Pull request by rdeheele**
see https://github.com/odoo/odoo/pull/5562

### CODE REVIEW

File: addons/hr_expense/views/report_expense.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
                                 <td>
                                     <span t-field="line.name"/> 
                                     <span t-field="line.description"/><br/>
-                                    <span t-field="line.analytic_account.complete_name" t-if="line.analytic_account"/>
+                                    <t t-if="line.analytic_account"><span t-field="line.analytic_account.complete_name"/></t>
```
Comment (pedrobaeza) [inline addons/hr_expense/views/report_expense.xml:47]: You can use directly `<span t-if...`

Comment (rdeheele) [inline addons/hr_expense/views/report_expense.xml:47]: you're right, in fact the original problem is the order! see odoo#5562

**Comment by rdeheele** [comment]
merged here: https://github.com/OCA/OCB/commit/66268842558c904348428c100159049339243845
I close.

**Comment by pedrobaeza** [comment]
Good to see that it has been solved, but I'm also worried about the statement: "Attributes order in QWeb matters".

**Pull request closed without merging**